scripts/scale/create_and_setup_instruction_following_project.py

```diff
@@ -0,0 +1,91 @@
+import argparse
+import json
+from scale_utils import get_scale_client
+from scaleapi.tasks import TaskType
+from scaleapi.exceptions import ScaleDuplicateResource
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--project_name", type=str, help="Name of the project to create")
+parser.add_argument(
+    "--credentials_path", type=str, default="prod_env/credentials.conf", help="Path to the credentials file"
+)
+args = parser.parse_args()
+project_name = args.project_name
+client = get_scale_client(args.credentials_path)
+
+print("\nGetting project...")
+try:
+    print(f"Trying to create project {project_name} ...")
+    project = client.create_project(
```

Comment:
I don't quite understand why we need to create a project instead of reusing an existing project. Let's discuss this in person.

Reply:
Today's production batch is in the old project, so we won't use this script today.

Reply:
Discussed in person^